modules/packages/LinearAlgebra.chpl

```diff
@@ -927,6 +927,25 @@ proc trace(A: [?D] ?eltType) {
   return trace;
 }
 
+proc kroneckerProduct(A: [?ADom] ?eltType, B: [?BDom] eltType) {
```

Comment:
I'd prefer a less verbose procedure name like `kron()`. It seems that several other linear algebra libraries made a similar decision in their interface.

Reply:
Ok